reasoner_pydantic/message.py

```diff
@@ -206,12 +206,9 @@ class Config:
 class Response(BaseModel):
     """Response."""
 
-    message: Message = Field(
-        ...,
-        title="message",
-    )
+    message: Message = Field(..., title="message", nullable=False)
 
-    logs: Optional[HashableSequence[LogEntry]] = Field(None, nullable=True)
+    logs: Optional[HashableSequence[LogEntry]] = Field(..., nullable=False)
```

Comment:
Should this one still be Optional?

Reply:
`logs` is only required for `AsyncQueryStatusResponse`, but not for `Response`.